projects/plugins/jetpack/tests/action-test-coverage.sh

```diff
@@ -13,11 +13,11 @@ done
 
 declare -A TESTS
 TESTS=()
-TESTS[php-normal]="php -dpcov.directory=. \"$(command -v phpunit)\" --coverage-clover \"$COVERAGE_DIR/backend/clover.xml\""
-TESTS[php-lfs]="WP_TESTS_CONFIG_FILE_PATH=\"${WP_TESTS_CONFIG_FILE_PATH%.php}.lfs.php\" LEGACY_FULL_SYNC=1 php -dpcov.directory=. \"$(command -v phpunit)\" --group=legacy-full-sync --coverage-clover \"$COVERAGE_DIR/legacy-sync/clover.xml\""
-TESTS[php-multisite]="WP_TESTS_CONFIG_FILE_PATH=\"${WP_TESTS_CONFIG_FILE_PATH%.php}.multi.php\" WP_MULTISITE=1 php -dpcov.directory=. \"$(command -v phpunit)\" -c tests/php.multisite.xml --coverage-clover \"$COVERAGE_DIR/multisite/clover.xml\""
-TESTS[client]="pnpm run test-client --coverage --collectCoverageFrom='_inc/client/state/**/*.js' --coverageDirectory=\"$COVERAGE_DIR/client\" --coverageReporters=clover"
-TESTS[gui]="pnpm run test-gui --coverage --collectCoverageFrom='_inc/client/state/**/*.js' --coverageDirectory=\"$COVERAGE_DIR/client\" --coverageReporters=clover"
-TESTS[extensions]="pnpm run test-extensions --coverage --collectCoverageFrom='_inc/client/state/**/*.js' --coverageDirectory=\"$COVERAGE_DIR/client\" --coverageReporters=clover"
+TESTS[php-normal]="php -dpcov.directory=. \"$(command -v phpunit)\" --coverage-php \"$COVERAGE_DIR/backend/php.cov\""
+TESTS[php-lfs]="WP_TESTS_CONFIG_FILE_PATH=\"${WP_TESTS_CONFIG_FILE_PATH%.php}.lfs.php\" LEGACY_FULL_SYNC=1 php -dpcov.directory=. \"$(command -v phpunit)\" --group=legacy-full-sync --coverage-php \"$COVERAGE_DIR/legacy-sync/php.cov\""
+TESTS[php-multisite]="WP_TESTS_CONFIG_FILE_PATH=\"${WP_TESTS_CONFIG_FILE_PATH%.php}.multi.php\" WP_MULTISITE=1 php -dpcov.directory=. \"$(command -v phpunit)\" -c tests/php.multisite.xml --coverage-php \"$COVERAGE_DIR/multisite/php.cov\""
+TESTS[client]="pnpm run test-client --coverage --coverageDirectory=\"$COVERAGE_DIR/client\" --coverageReporters=json"
+TESTS[gui]="pnpm run test-gui --coverage --coverageDirectory=\"$COVERAGE_DIR/client\" --coverageReporters=json"
+TESTS[extensions]="pnpm run test-extensions --coverage --coverageDirectory=\"$COVERAGE_DIR/client\" --coverageReporters=json"
```

Comment:
```suggestion
TESTS[gui]="pnpm run test-gui --coverage --coverageDirectory=\"$COVERAGE_DIR/gui\" --coverageReporters=json"
TESTS[extensions]="pnpm run test-extensions --coverage --coverageDirectory=\"$COVERAGE_DIR/extensions\" --coverageReporters=json"
```

Reply:
Looks like we got the wrong dir names here.

Reply:
Good catch